cis_v140/docs/cis_v140_4_10.md

```diff
@@ -0,0 +1,19 @@
+## Description
+
+Publish Domain-Based Message Authentication, Reporting and Conformance (DMARC) records for each Exchange Online Accepted Domain.
+
+## Remediation
+
+### From Console
+
+To add DMARC records, use the following steps:
+
+1. For each Exchange Online Accepted Domain, add the following record to DNS:
+
+```
+  Record:  _dmarc.domain1.com
+  Type:  TXT
+  Value:  v=DMARC1; p=none;
```

Comment:
Do we need to add space here? What we have done in other docs?
```suggestion
Record:  _dmarc.domain1.com
Type:  TXT
Value:  v=DMARC1; p=none;
```
Can you also check the other docs?

Reply:
Yes